bots/images/scripts/openshift.setup

```diff
@@ -2,6 +2,28 @@
 
 set -eux
 
+# Wait for x for many minutes
+function wait() {
+    for i in $(seq 1 100); do
+        if eval "$@"; then
+            return 0
+        fi
+        sleep 6
+    done
+    exit 6
+}
+
+function docker_images_has() {
+    docker images | tr -s ' ' | cut -d ' ' --output-delimiter=: -f1,2 | grep -q "$1"
```

Comment:
`docker images --format "{{.Repository}}:{{.Tag}}"` would be a bit more robust. You even already use that further down :-)